totalopenstation/formats/leica_gsi.py

```diff
@@ -325,11 +327,11 @@ def raw_line(self):
         '''
 
         points = []
-        ldata = len(self.line[0].split()[0].lstrip('*')[7:])
+        ldata = len(self.rows[0].split()[0].lstrip('*')[7:])
```

Comment:
I don't remember what this line does, perhaps we can add a comment to explain or give the variable a more explicit name?

Reply:
This variable is used to get the block code length (8 or 16).
It is not used anymore in points method but it is in the raw_line one.
I will drop it in points method.